libyul/backends/wasm/WasmCodeTransform.h

```diff
@@ -85,6 +85,8 @@ class WasmCodeTransform
 
 	wasm::FunctionDefinition translateFunction(yul::FunctionDefinition const& _funDef);
 
+	void importBuiltIn(BuiltinFunction const* _builtin, std::string const& _module, std::string const& _name);
```

Comment:
Maybe add a comment? import from where?

Reply:
Added a comment. Not sure whether it's easily understandable.